Tasks/ServiceFabricUpdateAppVersions/Tests/Update-ApplicationVersions.OldManifestNotFound.ps1

```diff
@@ -0,0 +1,4 @@
+[CmdletBinding()]
+param()
+
+. "$PSScriptRoot\Test-ApplicationVersions.ps1" PreviousPackageNoManifest
```

Comment:
I am. Look at line 57 in Test-ApplicationVersions.ps1(called from this test) wher we are validating that new application manifest has updated version.